apps/hobbies-helsinki/README.md

```diff
@@ -307,7 +307,7 @@ The "main events search" component is used in the search page at `/search`.
 | location          | []                                       | x            | -        | Filter by location / venue / place (source: LinkedEvents)                                     |
 | keywordAnd        | []                                       | x            | -        | Used for "Only children events" when it will be set to "yso:p4354"                            |
 | keywordOrSet2     | []                                       | x            | -        | A list of keywords (e.g categories or premapped categories)                                   |
-| keywordOrSet3     | []                                       | x            | -        | A list of keywords (e.g hobby types or premapped hobby types)                                 |
+| keywordOrSet3     | []                                       | x            | -        | A list of keywords                                                                            |
```

Comment:
Is the `keywordOrSet3` used at all? I think it should be removed from this table.

Reply:
https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/829/files#r2367141906

Reply:
It's better to remove it completely.

Reply:
Removed in [force push](https://github.com/City-of-Helsinki/events-helsinki-monorepo/compare/0c4b93bf7a97cfcf9931e26e574277fea7a8bcac..c3d7751230097612c20b6a981ca08511b1826dd9).